### ISSUE TITLE
Could not find io.kommunicate.sdk:kommunicateui:2.8.2. on expo run:android

### ISSUE THREAD

**Issue by antonio-silencio** [Kommunicate-io/Kommunicate-React-Native-SDK#57]
We are using the sdk with Expo.
On ios seems to works well. 
On android we can't build. 

running **expo run:android**

**FAILURE: Build failed with an exception.**

* What went wrong:
Could not determine the dependencies of task ':app:processDebugResources'.
Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
**Could not find io.kommunicate.sdk:kommunicateui:2.8.2.**
   
we have installed the last version -> "react-native-kommunicate-chat": "^2.3.0",

**Comment by Sathyan-KM** [comment]
Hi Antonio, 
Is your created through Expo Cli ?

**Comment by antonio-silencio** [comment]
yes. we are using expo-cli. we are not using expo-go but expo-dev-client.
